### PR TITLE
Rename cli binary from cfn-compose to cfnc

### PR DESCRIPTION
**Pull request by omegazyadav**
##### SUMMARY
It would be easier to use binary when it is short and more generic. This PR comprises of refactoring the existing setup to move the CLI binary from `cfn-compose` to `cfnc`. 

##### ISSUE TYPE
- Cleanups & Refactoring

##### COMPONENT NAME
- CLI Binary 

##### Screenshots
Screenshot from 2022-12-27 21-10-36

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,48 +34,48 @@ Available Commands:
   help        Help about any command
 
 Flags:
-  -c, --config string     File path to compose file (default "cfn-compose.yml")
```
Comment (rbalman) [inline README.md:37]: @omegazyadav  let's rename `cfnc.yml` back to `cfn-compose.yml` as we are only changing the name of the cli. Also rename this file `examples/ec2-sg/cfnc.yml`

**Comment by rbalman** [comment]
@omegazyadav  please do add screenshots of tests/validation of this change.

File: README.md (hunk 1 above)
Comment (omegazyadav) [reply README.md:37]: @rbalman I have reverted the corresponding changes. Thanks.

**Comment by omegazyadav** [comment]
Validating the `cfnc` binary to deploy the stack with dry run enabled. 
cc: @rbalman 

Screenshot from 2022-12-27 21-14-20

File: README.md [hunk 2]
```diff
@@ -168,47 +168,47 @@ Binary is available for Linux, Windows and Mac OS (amd64 and arm64). Download th
 
 Linux:
 ```
-curl -sSLO https://github.com/rbalman/cfn-compose/releases/download/v0.0.1-beta/cfn-compose-v0.0.1-beta-linux-amd64.tar.gz
+curl -sSLO https://github.com/rbalman/cfn-compose/releases/download/v0.0.1-beta/cfnc-v0.0.1-beta-linux-amd64.tar.gz
```
Comment (rbalman) [inline README.md:171]: Let's update version to 0.0.2-beta for now. We will have to make it dynamic in our future releases.
```suggestion
curl -sSLO https://github.com/rbalman/cfn-compose/releases/download/v0.0.1-beta/cfnc-v0.0.2-beta-linux-amd64.tar.gz
```

Comment (omegazyadav) [reply README.md]: Changes has been applied, Thanks.

**Review by rbalman — APPROVED** [review]
Thank you @omegazyadav  for your effort. Happy to merge this, finally we will have short and handy name.

**Pull request merged**